scripts/check-queries.lua

```diff
@@ -46,8 +46,8 @@ end
 
 local function do_check()
   local timings = {}
-  local parsers = require("nvim-treesitter.info").installed_parsers()
   local queries = require "nvim-treesitter.query"
+  local parsers = #_G.arg > 0 and { unpack(_G.arg) } or require("nvim-treesitter.info").installed_parsers()
```

Comment:
```suggestion
  local parsers = #_G.arg > 0 and _G.arg or require("nvim-treesitter.info").installed_parsers()
```
Would this not work?

Reply:
No, because `_G.arg[0]` is the script name (which `unpack` drops).